pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -266,6 +267,13 @@ func NewNodeLifecycleController(
 
 	eventBroadcaster := record.NewBroadcaster()
 	recorder := eventBroadcaster.NewRecorder(scheme.Scheme, v1.EventSource{Component: "node-controller"})
+	eventBroadcaster.StartLogging(glog.Infof)
+
+	glog.Infof("Sending events to api server.")
+	eventBroadcaster.StartRecordingToSink(
+		&v1core.EventSinkImpl{
+			Interface: v1core.New(kubeClient.CoreV1().RESTClient()).Events(""),
+		})
 
 	if kubeClient != nil && kubeClient.CoreV1().RESTClient().GetRateLimiter() != nil {
```

Comment:
we do not need to check kubeClient again, I'm ok to handle it in a separated PR.

Reply:
@k82cn ok, i will create a new pull request